projects/plugins/social/tests/php/test-class-jetpack-social.php

```diff
@@ -44,6 +44,7 @@ public function test_publicize_module_is_activated_on_plugin_activation() {
 
 		$connection_manager = $this->createMock( Connection_Manager::class );
 		$connection_manager->method( 'is_connected' )->willReturn( true );
+		$connection_manager->method( 'has_connected_user' )->willReturn( true );
```

Comment:
Should we add an extra test to make sure some stuff only gets loaded/initialised when we have a user connection?

Reply:
Good idea. Added in 8905a2c